modules/fundamental/src/advisory/model/mod.rs

```diff
@@ -58,15 +58,19 @@ pub struct AdvisoryHead {
 impl AdvisoryHead {
     pub async fn from_advisory(
         entity: &advisory::Model,
-        issuer: Option<organization::Model>,
+        issuer: Option<Option<organization::Model>>,
```

Comment:
This can't be right, can it? What behavior is it providing?

Reply:
It is in fact!

That method can take an organization from a previous query (which is `Option<org::Model>`) or can be deferred to let it do the lookup, if no caller has already queried it.

Therefore...

`None` -> I've not asked, please do it for me.
`Some(None)` -> I did the lookup, there isn't one, don't bother doing the query again.
`Some(Some(org_model))` -> I did the lookup, here it is, don't bother doing the query again.